src/main/java/edu/spingdata/hw14/controller/LibraryRestController.java

```diff
@@ -0,0 +1,90 @@
+package edu.spingdata.hw14.controller;
+
+import edu.spingdata.hw14.model.Author;
+import edu.spingdata.hw14.model.Book;
+import edu.spingdata.hw14.service.IPersonService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@AllArgsConstructor
+@RestController
+public class LibraryRestController {
+
+    IPersonService iPersonService;
```

Comment:
According to conventions, you should call your service interfaces like PersonService and its implementation like PersonServiceImpl.